src/components/plots/abstractPlot.ts

```diff
@@ -16,6 +16,12 @@ export module Plot {
     datasetKey: string
   }
 
+  export type PlotData = {
+    data: any[];
+    pixelPoints: Point[];
```

Comment:
`pixelPositions`? That's what it's named in `HoverData`. "`pixels`" would also work.